packages/core/src/index-deprecated.ts

```diff
@@ -159,7 +159,11 @@ export {
     nativePseudoClasses,
     nativePseudoElements,
 } from './native-reserved-lists';
-export { noCollisionNamespace, packageNamespaceFactory } from './resolve-namespace-factories';
+export {
+    noCollisionNamespace,
+    packageNamespaceFactory,
+    defaultBuildNamespace,
+} from './resolve-namespace-factories';
```

Comment:
`noCollisionNamespace` is exported, the fact that it appears here is a mistake, and `packageNamespaceFactory` wasn't in the known used list.

We can export them in v5 (`noCollisionNamespace` is already there). It would be nice to have them under identical prefix, "namespace", maybe?